### PR TITLE
VDS-2564: Upload import file

### PR DESCRIPTION
**Pull request by ilyawzrd**
Description: 

--

QA-test: VDS-2566

Demo-test: VDS-2567

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.DescriptionExportImport_1.0.0-pr-3-b3a754c6.zip

### CODE REVIEW

File: src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs [hunk 1]
```diff
@@ -6,34 +6,83 @@ namespace VirtoCommerce.DescriptionExportImportModule.Core
...
+
+            public const string ExceedingMaxLength = "exceeding-max-length";
+
+            public const string ArrayValuesExceedingMaxLength = "array-values-exceeding-max-length";
+
+            public const string InvalidValue = "invalid-value";
+
+            public const string NotUniqueValue = "not-unique-value";
+
+            public const string CountryNameAndCodeDoesntMatch = "country-name-and-code-doesnt-match";
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs]: Remove this error code

File: src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs [hunk 2]
```diff
@@ -6,34 +6,83 @@ namespace VirtoCommerce.DescriptionExportImportModule.Core
...
+        }
+
         public static class Security
         {
             public static class Permissions
             {
-                public static string[] AllPermissions { get; } = { };
+                public const string ImportAccess = "description:import";
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs]: Maybe something like "product:description:import"

File: src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj [hunk 3]
```diff
@@ -33,6 +33,7 @@
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.23.0" />
+        <PackageReference Include="VirtoCommerce.CatalogModule.Core" Version="3.59.0" />
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj]: Remove. It is in the Core already.

**Review by trueboroda — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs]: done

File: src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DescriptionExportImportModule.Core/ModuleConstants.cs]: done

File: src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj (hunk 3 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DescriptionExportImportModule.Data/VirtoCommerce.DescriptionExportImportModule.Data.csproj]: Done. Moved to Core project

**Review by trueboroda — APPROVED** [review]

**Pull request merged**